src/utils/internationalization/locales/en.json

```diff
@@ -0,0 +1,3 @@
+{
+	"Welcome to barefoot nomad": "Welcome to barefoot nomad"
```

Comment:
Fix the new line issue at the end of this file.
Also, there is an indent difference on this line between this file and [fr.json](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/ft-internationalization-174797453/src/utils/internationalization/locales/fr.json) or [kin.json](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/ft-internationalization-174797453/src/utils/internationalization/locales/kin.json)

Reply:
okay i am going to work on it

Reply:
:+1: